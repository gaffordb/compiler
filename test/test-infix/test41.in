let x : {bool} = {} : bool in
cdr (#f :: (#t :: (#f :: (#t :: x))))
