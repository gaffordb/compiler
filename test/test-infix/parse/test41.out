(let x = {   } : bool in
(cdr (:: #f(:: #t(:: #f(:: #tx))))))
